src/mongo/db/db.cpp

```diff
@@ -309,6 +309,27 @@ ExitCode _initAndListen(int listenPort) {
         l << (is32bit ? " 32" : " 64") << "-bit host=" << getHostNameCached() << endl;
```

Comment:
Những

Reply:
Can you explain what is your intention @anhems with this line?
Seems completely unrelated to this PR,

Best,
Miguel